Application/EdFi.Ods.Api/Helpers/AssemblyLoaderHelper.cs

```diff
@@ -146,6 +147,33 @@ public static IEnumerable<string> FindPluginAssemblies(string pluginFolder, bool
                 yield break;
             }
 
+            var apiModelFiles = Directory.GetFiles(pluginFolder, "ApiModel-EXTENSION.json", SearchOption.AllDirectories);
+
+            var physicalNames = new List<string>();
+
+            foreach (var apiModelFilePath in apiModelFiles)
+            {
+                var myJsonString = File.ReadAllText(apiModelFilePath);
+
+                var myJsonObject = JObject.Parse(myJsonString);
+
+                JToken physicalNameJToken = myJsonObject.SelectToken("$.schemaDefinition.physicalName");
+
+                if (physicalNameJToken != null)
+                {
+                    physicalNames.Add(physicalNameJToken.Value<string>());
+                }
+            }
+
+            var duplicatePhysicalName = physicalNames.GroupBy(x => x)
+                .Where(group => group.Count() > 1)
+                .Select(group => group.Key).ToList();
+
+            if (duplicatePhysicalName.Any())
+            {
+                throw new Exception($"Plugin folder has '{string.Join(",", duplicatePhysicalName)}' duplicate schema name .");

```

Comment:
Sorry I realize now my original suggestion about the error message was ambiguous. I think this message should be more helpful and should inform the user of a few things:
- these duplicate extension names have been found
- these are the names(probably the plugin's folder name) of the conflicting plugins
- that in order to get it working again they should remove one of the conflicting plugins and restart the api

My other concern is that we are not logging the error and only throwing. From a developer perspective throwing with the error message is enough because we have access to the console, but with a deployed api it is probably more useful if this error is logged so we can see it in the log file. I would still keep the throw to stop startup. Restating my original suggestion:
- log the error using the logger with the message
- throw to stop the execution, no message needed

Reply:
I fixed it now